project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.1.6
```

Comment:
Why would we choose such an old version? The latest release is `1.4.5`

Reply:
I'll bump it to what we use in mithril.

Reply:
They don't need to be in sync. This works, so does the Mithril version or the latest version... was just curious as to why an old version.